src/libraries/System.Formats.Tar/src/System/Formats/Tar/PaxTarEntry.cs

```diff
@@ -50,7 +49,7 @@ internal PaxTarEntry(TarHeader header, TarReader readerOfOrigin)
         /// </list>
         /// </remarks>
         public PaxTarEntry(TarEntryType entryType, string entryName)
-            : base(entryType, entryName, TarEntryFormat.Pax)
+            : this(entryType, entryName, extendedAttributes: new Dictionary<string, string>())
```

Comment:
We are double allocating a Dictionary here. It would be better if we passed in `null`, or some sentinel, to indicate an empty set of attributes.

Reply:
Good point. I won't call `this(...)` for this constructor. Instead, I'll use similar logic to the constructor below, skipping the check for a non-null dictionary and not setting a value for it.

Reply:
Actually, I have to initialize the dictionary here, because I need to set an `atime` and a `ctime` below. So I will still do what I said in the previous comment of using separate logic, but I will only allocate the dictionary once.